beacon-chain/rpc/beacon/validators.go

```diff
@@ -802,15 +802,25 @@ func (bs *Server) GetValidatorPerformance(
 	correctlyVotedHead := make([]bool, 0, reqPubKeysCount)
 	missingValidators := make([][]byte, 0, reqPubKeysCount)
 
-	headState, err := bs.HeadFetcher.HeadState(ctx)
-	if err != nil {
-		return nil, status.Error(codes.Internal, "Could not get head state")
-	}
+    headState, err := bs.HeadFetcher.HeadState(ctx)
```

Comment:
please use `gofmt` here the indentation is off.

Reply:
Okay